bootstrap.sh

```diff
@@ -14,12 +14,12 @@ USE baboon;
 create table users(username VARCHAR(25) NOT NULL, password VARCHAR(100) NOT NULL, creation_date TIMESTAMP DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP, PRIMARY KEY(username));
 MYSQL_SCRIPT
 
-if [ $TRAVIS ]
-then
-   my_path=$TRAVIS_BUILD_DIR
-else
-   my_path='/vagrant'
-fi
+#if [ $TRAVIS ]
+#then
+#   my_path=$TRAVIS_BUILD_DIR
+#else
+#   my_path='/vagrant'
+#fi
```

Comment:
Remove the comments please. It clutters the code, which reduces it maintainability.